libnavigation-core/src/main/java/com/mapbox/navigation/core/MapboxDistanceFormatter.kt

```diff
@@ -8,69 +8,64 @@ import android.text.style.RelativeSizeSpan
 import android.text.style.StyleSpan
 import com.mapbox.navigation.base.formatter.DistanceFormatter
 import com.mapbox.navigation.base.typedef.IMPERIAL
-import com.mapbox.navigation.base.typedef.METRIC
-import com.mapbox.navigation.base.typedef.RoundingIncrement
 import com.mapbox.navigation.base.typedef.VoiceUnit
 import com.mapbox.navigation.trip.notification.R
 import com.mapbox.navigation.trip.notification.utils.extensions.getUnitTypeForLocale
 import com.mapbox.navigation.utils.extensions.inferDeviceLocale
-import com.mapbox.turf.TurfConstants.UNIT_FEET
-import com.mapbox.turf.TurfConstants.UNIT_KILOMETERS
-import com.mapbox.turf.TurfConstants.UNIT_METERS
-import com.mapbox.turf.TurfConstants.UNIT_MILES
+import com.mapbox.turf.TurfConstants
 import com.mapbox.turf.TurfConversion
 import java.text.NumberFormat
 import java.util.Locale
+import kotlin.math.roundToInt
 
-/**
- * Creates an instance of DistanceFormatter, which can format distances in meters
- * based on a language format and unit type.
- *
- *
- * This constructor will infer device language and unit type using the device locale.
- *
- * @param language for which language
- * @param unitType to use, or UNDEFINED to use default for locale country
- * @param context from which to get localized strings from
- * @param roundingIncrement increment by which to round small distances
- */
-class MapboxDistanceFormatter(
-    private val context: Context,
-    language: String?,
-    @VoiceUnit unitType: String,
-    @RoundingIncrement private val roundingIncrement: Int
-) : DistanceFormatter {
-
-    private val unitStrings = hashMapOf<String, String>(
-        UNIT_KILOMETERS to context.getString(R.string.kilometers),
-        UNIT_METERS to context.getString(R.string.meters),
-        UNIT_MILES to context.getString(R.string.miles),
-        UNIT_FEET to context.getString(R.string.feet)
-    )
-    private val largeUnit = when (IMPERIAL == unitType) {
-        true -> UNIT_MILES
-        false -> UNIT_KILOMETERS
+class MapboxDistanceFormatter(private val context: Context, private val unitType: String, private val roundingIncrement: Int) : DistanceFormatter {
+
+    // These values are in meters.
```

Comment:
What about adding that in the naming and remove this comment?

Also, should these be moved to the `companion object` and make them `const`s?

```kotlin
companion object {
    private const val SMALL_DISTANCE_UPPER_THRESHOLD_IN_METERS = 400.0
    private const val MEDIUM_DISTANCE_LOWER_THRESHOLD_IN_METERS = 401.0
    private const val MEDIUM_DISTANCE_UPPER_THRESHOLD_IN_METERS = 10000.0
    @JvmStatic
    fun builder(context: Context): Builder = Builder(context)
}
```